test/Testing.py

```diff
@@ -60,3 +60,16 @@ def get_tested_spec_package(name):
     candidates = list(get_tested_path(name).parent.glob(filename))
     assert len(candidates) == 1
     return FakePkg(candidates[0])
+
+
+def get_tested_mock_package(files=None, real_files=None, header=None):
+    mockPkg = FakePkg('mockPkg')
+    if files is not None:
+        for path, file in files.items():
+            if file.get('create_dirs'):
+                for i in PurePath(path).parents[:file.get('include_dirs', -1)]:
+                    mockPkg.add_dir(str(i))
+            mockPkg.add_file_with_content(path, file.get('content'), real_files=real_files)
```

Comment:
Nitpick: This is getting complicated, maybe it's worth to move this code to a method in the `FakePkg` class, or to another helper function to reduce the if-for nesting. It's just 5 lines and it's not too complex right now, so we could keep the code as it is, but if we find a way to make it more readable, it could be an improvement.